frontend/components/App/Onboarding/index.tsx

```diff
@@ -36,19 +36,19 @@ const SlickButtonFix = ({ children, ...props }) => (
   </button>
 );
 
-const PrevArrow = ({ currentSlide, slideCount, children, ...props }: any) => (
```

Comment:
this change does not seem correct, I think it was done like this for a reason. cc @eddsaura

Reply:
I think it does. In fact, I think that the previous implementation was incorrect since those props are not used in the component; and if we [see the documentation](https://react-slick.neostack.com/docs/example/custom-arrows), it does not indicate that those props have to be declared.

Reply:
But it involves an error on browsers when trying to render props that are not part of the html spec for that kind of tag. Did you check those? @eddsaura fixed that a while ago.

Reply:
Fixed and documented 👍

Reply:
Maybe you will need to silence those eslint errors if you restore the function.